src/components/PPBlock/index.tsx

```diff
@@ -9,17 +9,29 @@ export type PPCardProps = {
 };
 
 const PPBlock: React.FC<PPCardProps> = (props) => {
-  return (
-    <div className={styles.ppcard} style={props.style}>
-      <Row className={styles.titleRow}>
-        <Title className={styles.title}>{props.title}</Title>
-      </Row>
-      <Row style={{ marginTop: 26 }}>
-        <Col span={24} style={{ paddingLeft: 30, paddingRight: 30, textAlign: 'center' }}>
-          {props.children}
-        </Col>
-      </Row>
-    </div>
-  );
+  if (props.title) {
+    return (
+      <div className={styles.ppcard} style={props.style}>
+        <Row className={styles.titleRow}>
```

Comment:
Why not simply add a `display: none` to this row when not passing props.title like this:

```typescript
<Row className={styles.titleRow} style={{display: props.title ? undefined : 'none'}}>
```

Reply:
i see, i dont know there was `display`